moveit_planners/chomp/chomp_interface/test/rrbot_chomp_planning_pipeline.launch.xml

```diff
@@ -0,0 +1,14 @@
+<launch>
+	<!-- CHOMP Plugin for MoveIt! -->
+	<arg name="planning_plugin" value="chomp_interface/CHOMPPlanner" />
+
+	<arg name="start_state_max_bounds_error" value="0.1" />
+
+	<param name="planning_plugin" value="$(arg planning_plugin)" />
+	<param name="start_state_max_bounds_error" value="$(arg start_state_max_bounds_error)" />
+
+	<param name="collision_detector" value="Hybrid" />
+
+	<rosparam command="load" file="$(find moveit_planners_chomp)/test/chomp_planning.yaml"/>	
```

Comment:
At some point we should probably put these parameters in a namespace, right now its all cluttered into /move_group

Reply:
We should add this to a CHOMP issue list and work on it one issue at a time.